modules/adding-new-extension-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="adding-new-extension-dynamic-plugin_{context}"]
+= Adding a new extension to your plug-in
+You can add different extensions to your plug-in that are loaded at runtime.
+
+[id="adding-tab-to-pods-page"]
+== Adding a tab to the pods page
+You can add a tab to the *Pod Details* page.
+
+.Procedure
+
+. Edit the `console-extensions.json` file.
++
+[source,json]
+
+----
+}
+    "type": "console.page/resource/tab",
+    "properties": {
+        "name": "Example Tab",
+        "href": "example",
+        "model": {
+            "group": "core",
+            "version": "v1",
+            "kind": "Pod"
+        },
+        "component": { "$codeRef": "ExampleTab" }
+}
+----
+
+. Edit the `package.json` file.
++
+[source,json]
+
+----
+        "displayName": "OpenShift Console Plugin Template",
+        "description": "Template project for OpenShift console plugins. Edit package.json to change this message and the plugin name.",
+        "exposedModules": {
+            "ExamplePage": "./components/ExamplePage",
+            "ExampleTab": "./components/ExampleTab"
+        },
+        "dependencies": {
+            "@console/pluginAPI": "*"
```

Comment:
Since we're adding a new `exposedModule`, I would probably just include that part of the JSON. We might move this below creating `src/components/ExamplePage.tsx`. (Generally you create the module first, then add it here.)

```suggestion
        "exposedModules": {
            "ExamplePage": "./components/ExamplePage",
            "ExampleTab": "./components/ExampleTab"
        },
```